tests/test_smoke.py

```diff
@@ -66,9 +67,9 @@
 SCP_GPU_V100 = '--gpus V100-32GB'
 
 storage_setup_commands = [
-    'touch ~/tmpfile', 'mkdir -p ~/tmp-workdir',
+    'touch ~/tmpfile', 'mkdir -p ~/tmp-workdir', 'mkdir -p ~/tmp-workdir-2',
```

Comment:
sry why are we doing this?

Reply:
If you refer to `tests/test_yamls/test_storage_mounting.yaml.j2`, `tmp-workdir` is used to test `MOUNT` mode, so we need to creat an additional directory to test `CSYNC` mode.